problem_builder/dashboard.py

```diff
@@ -207,8 +222,23 @@ class DashboardBlock(StudioEditableXBlockMixin, XBlock):
         ),
         scope=Scope.content,
     )
+    average_label = String(
+        display_name=_("Label for average value"),
+        default=_("Average"),
+        help=_("Label to be shown for calculated average"),
+        scope=Scope.content,
+    )
+    show_numbers = Boolean(
+        display_name=_("Display numeric values"),
```

Comment:
I think this should just be "Display values", since the values *can* be non-numeric.